skins/oasis/css/core/ads.scss

```diff
@@ -14,6 +14,10 @@
 	display: none !important;
 }
 
+.source-point .wikia-ad.default-height {
```

Comment:
It's used only in "old" slots divs, but I guess we can do the same thing by removing those classes in JS code instead of adding .source-point to body, what do you think?